profiling/nvtx-connector/CMakeLists.txt

```
@@ -1,4 +1,4 @@
 find_package(CUDAToolkit REQUIRED)
 kp_add_library(kp_nvtx_connector kp_nvtx_connector.cpp)
 
-target_link_libraries(kp_nvtx_connector CUDA::nvToolsExt)
+target_link_libraries(kp_nvtx_connector CUDA::nvtx3)
```

We could try something like
```suggestion
if (TARGET CUDA::nvtx3)
  target_link_libraries(kp_nvtx_connector CUDA::nvtx3)
else()
  target_link_libraries(kp_nvtx_connector CUDA::nvToolsExt)
endif()
```
and try to see if the `nvToolsExt` target still exists with CUDA 12.9 and if so if it provides the header used below.

---

Thanks for the code suggestion! However, unfortunately, the target `CUDA::nvToolsExt` is not defined as of Cuda 12.9. I've just pushed a version with a manually created imported target. Let's see.